build.gradle

```diff
@@ -28,72 +25,80 @@ allprojects {
     }
 }
 
+ext {
+    buildVersions = [
+            minSdk    : 16,
+            targetSdk : 27,
+            compileSdk: 27,
+            buildTools: "27.0.2"
+    ]
+
+    libsVersions = [
+            kotlin    : kotlin_version,
+            support   : "27.1.0",
+            constraint: "1.0.2",
+            picasso   : "2.5.2",
+            glide     : "4.6.1",
+            gson      : "2.8.2",
+            canarinho : "1.2.0",
+            lottie    : "2.5.0-rc1"
+    ]
+
+    testLibsVersions = [
+            jUnit        : "4.12",
+            espresso     : "3.0.1",
+            mockito      : "2.12.0",
+            mockitoKotlin: "1.5.0",
+            roboeletric  : "3.6.1"
+    ]
+
+    lintersVersions = [
+            ktlint: "0.19.0",
```

Comment:
this should be 0.20.0 or else it breaks ktlint

Reply:
You're right!